services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -1224,6 +1258,21 @@ class HankeServiceITests : DatabaseTest() {
         )
     }
 
+    private fun assertFeaturePropertiesContainOnlyDesiredFields(
+        hanke: Hanke,
+        propertiesWanted: Map<String, Any?>
+    ) {
+        val alueet = hanke.alueet
+        assertThat(alueet.size).isEqualTo(2)
```

Comment:
This will throw an exception from this row. Creating a custom validator instead of `assertFeaturePropertiesContainOnlyDesiredFields` function would throw the exception from the actual test.

Reply:
Alueiden määrän tarkistaminen ei oikein tunnu sopivan tähän funktioon.